feature-libs/quote/components/list/quote-list.component.ts

```diff
@@ -0,0 +1,132 @@
+  protected cxDatePipe = inject(CxDatePipe);
+  protected breakpointService = inject(BreakpointService);
+
+  sorts = this.quoteListComponentService.sortOptions;
+  sortLabels$ = this.quoteListComponentService.sortLabels$;
+  quotesState$ = this.quoteListComponentService.quotesState$;
+  dateFormat: string = 'MMMM d, YYYY h:mm aa';
+  iconTypes = ICON_TYPE;
+
+  constructor() {
+    this.changePage(0);
+    this.changeSorting(this.quoteListComponentService.defaultSortOption);
+  }
+
+  /**
+   * Changes current sorting.
+   *
+   * @param sortCode - Identifies sort option that should be applied
+   */
+  changeSorting(sortCode: string): void {
+    this.quoteListComponentService.setSorting(sortCode);
+  }
+
+  /**
+   * Changes current page.
+   *
+   * @param page - Desired page number
+   */
+  changePage(page: number): void {
+    this.quoteListComponentService.setPage(page);
+  }
+
+  /**
+   * Retrieves the class name for the quote state. This class name is composed
+   * using 'quote-' as prefix and the last part of the status name in lower case
+   * (like e.g. draft for SELLER_DRAFT).
+   *
+   * @param state - quote state
+   * @returns class name corresponding to quote state.
+   */
+  getQuoteStateClass(state: QuoteState): string {
+    const stateAsString = state.toString();
+    const indexSeparator = stateAsString.indexOf('_');
+    //note: in case not found: indexSeparator is -1, lastPart will be stateAsString
+    const lastPart = stateAsString.substring(indexSeparator + 1);
+    return 'quote-' + lastPart.toLowerCase();
+  }
+
+  /**
+   * Retrieves an accessibility text for a row in the quote list.
+   *
+   * @param quote - quote
+   * @returns an accessibility text for a row in the quote list
+   */
+  getRowTitle(quote: Quote): string {
+    let translatedText = '';
+    translatedText += quote.name;
+    translatedText += ' ';
+    this.translationService
+      .translate('quote.header.overview.id')
+      .pipe(take(1))
+      .subscribe((text) => (translatedText += text));
+    translatedText += ': ';
+    translatedText += quote.code;
+    translatedText += ' ';
+    this.translationService
+      .translate('quote.header.overview.status')
+      .pipe(take(1))
+      .subscribe((text) => (translatedText += text));
+    translatedText += ': ';
+    this.translationService
+      .translate('quote.states.' + quote.state)
+      .pipe(take(1))
+      .subscribe((text) => (translatedText += text));
+    translatedText += ' ';
+    this.translationService
+      .translate('quote.list.updated')
+      .pipe(take(1))
+      .subscribe((text) => (translatedText += text));
+    translatedText += ': ';
+    translatedText += this.cxDatePipe.transform(
+      quote?.updatedTime,
+      this.dateFormat
+    );
+    translatedText += ' ';
+    this.translationService
+      .translate('quote.list.clickableRow')
+      .pipe(take(1))
+      .subscribe((text) => (translatedText += text));
+    return translatedText;
+  }
```

Comment:
@Larisa-Staroverova

Reply:
Hi @Platonn, using this method we are trying to get the text for the row tooltip. Let me attach a screenshot.
As you could probably see from the screenshot, the tooltip contains a bunch of information that is read out by the screen reader.
Normally we follow your approach when we need a translation, but in this case it is not possible.
Please correct me if I'm wrong.
![image](https://github.com/SAP/spartacus/assets/61147963/69752178-d6a6-4eb3-afbd-940f2a705302)

Reply:
Hi @Platonn,
I have refactored the above mentioned method as we discussed it yesterday by using `combineLatest`

Reply:
Hi @Larisa-Staroverova 
I'm not sure I was understood well on the meeting.
I prefer the most a solution that uses `cxTranslate` pipe in HTML. I've mention the solution with `combineLatest` in TS only for completeness of the set of options to choose from. But it has a drawback, that we create a new observable during every render of this component.

Reply:
Instead of placing translation labels in TS, I meant putting them in HTML, as usual. For example:
```html
           <tr
              ...
              
              attr.title="
                {{ 'quote.list.name' | cxTranslate }}: {{ quote.name }}
                {{ 'quote.header.overview.id' | cxTranslate }}: {{ quote.code }}
                {{ 'quote.header.overview.status' | cxTranslate }}: {{
                  'quote.states.' + quote.state | cxTranslate
                }}
                {{ 'quote.list.updated' | cxTranslate }}: {{
                  quote?.updatedTime | cxDate: dateFormat
                }}
                {{ 'quote.list.clickableRow' | cxTranslate }}
              "
            >
```

Reply:
I got it.
Thanks, Kris a lot. 
I applied your latest feedback.